javascripts/imagetoscheme.html

```diff
@@ -102,6 +102,10 @@ <h1>Image to Scheme</h1>
       }
 
       on('output', (div, { image: c, resolution: RESOLUTION }) => {
+        
+        // if the RESOLUTION isn't a positive integer then set it to 3
+        RESOLUTION = (!(Number.isInteger(RESOLUTION) && RESOLUTION > 0)) ? 3 : RESOLUTION;
```

Comment:
you could flip the ternary around
```suggestion
        // if the RESOLUTION isn't a positive integer then set it to 3
        RESOLUTION = Number.isInteger(RESOLUTION) && RESOLUTION > 0 ? RESOLUTION : 3;
```

Reply:
you could also `return` from the function if the value is invalid so it doesn't change anything until the user types in a valid value

Reply:
oh i forgot that i could flip the ternary around because when i made it, it was originally an if statement lol